users/templates/signup.html

```diff
@@ -1,24 +1,29 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
-{% load rest_framework %}
 
 {% block content %}
 <h2>{% trans 'Sign up' %}</h2>
 
 <div class="modal-dialog" style="margin-bottom:0">
-  <form action="{% url 'signup' %}" method="post">
-      {% csrf_token %}
-      {% render_form serializer template_pack='rest_framework/vertical' %}
+    <form action="{% url 'signup' %}" method="post">
+        {% csrf_token %}
+        <div class="fieldWrapper">
+            <label for="{{ form.email.label }}">Email:</label>
```

Comment:
Please add translation to strings in this file or move it to enum too.

Reply:
Done